README.md

```diff
@@ -138,6 +138,8 @@ For clusters of more than 100 nodes, allocate at least
 
 These numbers are based on [scalability tests](https://github.com/kubernetes/kube-state-metrics/issues/124#issuecomment-318394185) at 30 pods per node.
 
+Note that if CPU limits are set too low, kube-state-metrics' internal queues will not be able to be worked off quickly enough, resulting in increased memory consumption as the queue length grows. If you experience problems resulting from high memory allocation, try increasing the CPU limits.
```

Comment:
`s/kube-state-metrics'/kube-state-metrics's`

Reply:
BTW, @while1malloc0 you need to sign the CLA before we can merge this PR. :)

Reply:
actually kube-state-metrics' is correct english grammar

Reply:
> actually kube-state-metrics' is correct english grammar

My bad. This is the second time that i think i should enhance my english skill, including wring and speak. :)